deploy/crds/apps.3scale.net_apimanagers_crd.yaml

```diff
@@ -41,19 +41,1353 @@ spec:
                   type: boolean
                 productionSpec:
                   properties:
+                    affinity:
```

Comment:
Definitely this CRD file is not human readable anymore. It is already too big. 

We have to trust on tooling